csrc/preprocess/transform/default_format_bundle.h

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) OpenMMLab. All rights reserved.
+
+#ifndef MMDEPLOY_DEFAULT_FORMAT_BUNDLE_H
+#define MMDEPLOY_DEFAULT_FORMAT_BUNDLE_H
+
+#include "core/tensor.h"
+#include "transform.h"
+
+namespace mmdeploy {
+/**
+ * It simplifies the pipeline of formatting common fields, including "img",
+ * "proposals", "gt_bboxes", "gt_labels", "gt_masks" and "gt_semantic_seg".
+ * These fields are formatted as follows.
+ *
+ *  - img: (1)transpose, (2)to tensor, (3)to DataContainer (stack=True)
+ *  - proposals: (1)to tensor, (2)to DataContainer
+ *  - gt_bboxes: (1)to tensor, (2)to DataContainer
+ *  - gt_bboxes_ignore: (1)to tensor, (2)to DataContainer
+ *  - gt_labels: (1)to tensor, (2)to DataContainer
+ *  - gt_masks: (1)to tensor, (2)to DataContainer (cpu_only=True)
+ *  - gt_semantic_seg: (1)unsqueeze dim-0 (2)to tensor, \
+ *                     (3)to DataContainer (stack=True)
+ *
+ */
+class MMDEPLOY_API DefaultFormatBundleImpl : public TransformImpl {
+ public:
+  DefaultFormatBundleImpl(const Value& args);
+  ~DefaultFormatBundleImpl() = default;
```

Comment:
miss `override`

Reply:
Shall we add `override` to all the constructor and destructor functions for the classes inherited from `Transform` and `TransformImpl`?

Reply:
It's a clang-tidy warning, not an error. The rest ones can be fixed in another PR